packages/devtools_app/lib/src/inspector/diagnostics.dart

```diff
@@ -209,14 +215,37 @@ class DiagnosticsNodeDescription extends StatelessWidget {
             textStyle.merge(inspector_text_styles.regularBold(colorScheme));
       }
 
-      children.add(Flexible(
+      children.add(Expanded(
         child: buildDescription(
           diagnostic.description,
           textStyle,
           colorScheme,
           isProperty: false,
         ),
       ));
+
+      if (errorText != null) {
+        children.add(
+          Expanded(
+            // Errors tend to be longer than widget classes, so allow them
+            // to take up a larger proportion of the space.
+            flex: 2,
```

Comment:
Would be better to let the Row layout itself know that the Description is flexible so does not need to use the full width it is offered.